ruby_programming/test_driven_development/rspec_introduction_lesson.md

```diff
@@ -1,26 +1,245 @@
 ### Introduction
-A brief summary about what this lesson is about and why the topics or concepts it covers are important.
+
+By now you've had some experience writing actual programs using Ruby.  Presumably, you've been making sure they work by running your script, manually setting up, entering individual inputs, and watching the output for correctness.
+
+Presumably, you've *also* suffered minor concussions from head-desking as you painstakingly repeated this cycle over, and over, and over again on the more complex projects (looking at you, Algorithms and Data Structures).
+
+For those of us who can't afford repeated hospital visits (and the few who can), there is good news: Ruby has a number of frameworks that can relieve some of the headaches of testing by simplifying and automating the aforementioned cycle. In this lesson, we'll introduce you to your new best friend, the RSpec testing framework.
 
 ### Learning Outcomes
-*Look through these now and then use them to test yourself after doing the assignment*
 
-what the student is expected to know or be able to do by the end of this lesson
+*Look through these now and use them to guide your learning. By the end of this lesson, expect to:*
+
+- Know what RSpec is
+- Know how to install RSpec
+- Understand basic RSpec syntax (`describe` / `it`)
```

Comment:
What do you think about splitting this out? 

* What is `describe` for
* What is `it` for

etc

Reply:
I think they'd work as bullets nested under "Understand basic RSpec syntax". Thoughts?

Reply:
Sounds good to me 👍